src/data/builder-tools.js

```diff
@@ -1138,6 +1138,22 @@ export const Showcases = [
     website: "https://github.com/SAIB-Inc/Cardano.Sync",
     getstarted: null,
     tags: ["net", "sdk", "chainindex"]
+  },
+  {
+    title: "Cardano Looking Glass",
+    description: "A visual blockchain explorer which supports CIP54 and CIP68 Smart NFTs.",
+    preview: require("./builder-tools/clg.png"),
+    website: "https://clg.wtf/",
+    getstarted: null,
+    tags: ["nft", "hosted"]
+  },
+  {
+    title: "NFT Playground",
+    description: "An integrated development environment for building CIP54-compliant Smart NFTs.",
+    preview: require("./builder-tools/nft-playground.png"),
+    website: "https://nft-playground.dev/",
+    getstarted: "https://nft-playground.dev/help",
+    tags: ["ide", "sdk", "hosted", "nft", "javascript"]
```

Comment:
you are making up tags / use them in a different way that suggested on https://developers.cardano.org/docs/portal-contribute/

Build breaks because of this.

Reply:
My apologies, very sloppy of me to have submitted this without testing the build - in my mind, I had tested it, so I'm assuming I probably ran `yarn build` in the wrong terminal window, and I successfully built some other project. 

Anyway, I have fixed the tag that was breaking the build - however I'm not sure what you mean by "in a different way to that suggested on the contribution page". As far as I can see the contribution page makes no mention of the tags or how to use them. Is there a tag in particular that I've added which you think is not suitable?

Reply:
@kieransimkin the individual tag descriptions are here: https://github.com/cardano-foundation/developer-portal/blob/staging/src/data/builder-tools.js
... i.e. this file in your local copy: `src/data/builder-tools.js`

@katomm I've also confirmed in a local build based on this PR branch that all tags are valid & it builds successfully. ✅